docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -49,6 +49,26 @@ To change the width of a column, use the `width` property available in `ColDef`.
 
 {{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
 
+## Column fluid width
+
+By default each column have a fixed width of 100 pixels but we can have fluid (responsive) columns by setting the `flex` propery avaibale in the `ColDef`.
+
+The `flex` propery accepts values between 0 and 1.
+
+The `flex` property works by dividing the remaining space in the grid among all flex columns in proportion to their `flex` value. For example, a grid with a total width of 500px that has three columns: the first with width: 200; the second with flex: 1; and third with flex: 0.5. The first column will be 200px wide, leaving 300px remaining. The column with flex: 1 has twice the size with flex: 0.5 which means that final sizes will be: 200px, 200px, 100px.
+
+The `flex` configuration does not work together with the `width` configuration. If you set both the `flex` and `width` inside the `ColDef` of column the `flex` will overwrite the `width`.
+
+The `flex` configuration does not work if the combined width of the columns that have `width` is more that the width of the grid itself. If that is the case a scroll bar will be visible and the columns that hava `flex` will default back to their base value of 100 pixels.
+
+{{"demo": "pages/components/data-grid/columns/ColumnFluidWidthGrid.js", "bg": "inline"}}
+
+<!--
```

Comment:
I think that we can remove these comments. We can include the benchmark in the related issue.

Reply:
These links are already in the ticket itself. I saw that we also added some benchmark links in the `resizing` section so that why I added these once but yes without them it looks cleaner ;)

Reply:
Definitely, I was hesitating about where it would be best to store these benchmarks. What you see in the resizing section is something I forgot to clean up :). I came to the conclusion that the list of a benchmark is living content. it would be too slow to edit them in GIT. It might also create issues if we translate the page.